lib/components/proposals/ProposalUI.jsx

```diff
@@ -478,60 +479,58 @@ const VoteButtons = (props) => {
 
   if (tx?.completed && !tx?.error && !tx?.cancelled) {
     return (
-      <TxText className='text-green'>
+      <TxText className='text-green ml-auto'>
         🎉 {t('successfullyVoted')} - {votingFor ? t('accept') : t('reject')} 🎉
       </TxText>
     )
   }
 
   if (tx?.inWallet && !tx?.cancelled && !tx?.error) {
-    return <TxText>{t('pleaseConfirmInYourWallet')}</TxText>
+    return <TxText className='ml-auto'>{t('pleaseConfirmInYourWallet')}</TxText>
   }
 
   if (tx?.sent) {
-    return <TxText>{t('waitingForConfirmations')}...</TxText>
+    return <TxText className='ml-auto'>{t('waitingForConfirmations')}...</TxText>
   }
 
   return (
-    <>
+    <div className='flex mt-2 justify-end'>
       {tx?.error && (
         <div className='text-red flex'>
           <FeatherIcon icon='alert-triangle' className='h-4 w-4 stroke-current my-auto mr-2' />
           <p>{t('errorWithTxPleaseTryAgain')}</p>
         </div>
       )}
-      <div className='h-10 xs:h-12'>
```

Comment:
I believe I added this so the UI wouldn't skip and jump around when you're loading content. It can be jarring to go from an empty div taking up 0 height to a content filled div taking up 100px of height.

Reply:
I removed it since it looks kind of silly with this big padding all the time. I don't think there's much of a delay, the buttons appear along with the usual page rendering. There was still a jump when going from no wallet -> wallet connected with the `h-10`. It's also adding ~45px to a card that will already have content so it's not a big of a surprise.

That being said, I think the loading states have been quite neglected and could be much better.

<img width="1027" alt="Screen Shot 2021-03-11 at 10 36 13 AM" src="https://user-images.githubusercontent.com/13701258/110812465-a19a3100-8255-11eb-9f12-dad87e148528.png">